### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by schtvr**
Using static vars for user info. Structure of profile page and its components established. Still needs system styling and backend util.

### CODE REVIEW

File: Client/App.tsx [hunk 1]
```diff
@@ -2,14 +2,16 @@ import React, {Profiler} from 'react';
 import {NavigationContainer} from '@react-navigation/native';
 import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
 import type {Node} from 'react';
+
+import HomeScreen from './screens/HomeScreen';
+import ProfileScreen from './screens/ProfileScreen';
+import SocialScreen from './screens/SocialScreen';
+import TrophyRoomScreen from './screens/TrophyRoomScreen';
+import SettingsScreen from './screens/SettingsScreen';
+
```
Comment (timdh94) [inline Client/App.tsx:11]: no spaces between imports

File: Client/Screens/ProfileScreen.tsx [hunk 2]
```diff
@@ -1,12 +1,87 @@
...
 };
 
+interface IUser {
+  userName: string,
+  profileUrl: string,
+  completedQuests: number,
+  curatedTrophies: any[],
+  activeQuests: any[],
+  questHistory: any[],
+  level: number,
```
Comment (timdh94) [inline Client/Screens/ProfileScreen.tsx:31]: place interfaces in a separate types folder

File: Client/Screens/ProfileScreen.tsx [hunk 3]
```diff
@@ -1,12 +1,87 @@
...
+  EXP: number,
+}
+
+
+
+export const user : IUser = {
```
Comment (timdh94) [inline Client/Screens/ProfileScreen.tsx:37]: should be user: IUser, why did you put that useless space there?

File: Client/Screens/ProfileScreen.tsx [hunk 4]
```diff
@@ -1,12 +1,87 @@
...
+  level: 3,
+  EXP: 280
+}
+
+const styles = StyleSheet.create({
+  pfp: {
+    flexDirection: 'row'
+  }
+})
+
```
Comment (timdh94) [inline Client/Screens/ProfileScreen.tsx:86]: style sheets in a separate file, for the love of GOD

File: Client/components/profile/QuestCard.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+export default QuestCard;
+/*
+
+    <Image
+      style={styles.image}
+      resizeMode="cover"
+      source={{ uri: quest.avatar }}
+    />
+*/
```
Comment (timdh94) [inline Client/components/profile/QuestCard.tsx:44]: no comments in pull requests

File: Client/components/profile/completedStats.tsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+import { user } from '../../screens/ProfileScreen';
+import CircularProgressBar from './progressCircle';
+
```
Comment (timdh94) [inline Client/components/profile/completedStats.tsx:5]: only one space here please

File: Client/components/profile/curatedTrophies.tsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+            source={imagePath}
+          />
+          <Text style={styles.trophyName}>{name}</Text>
+        </View>
+      )
+  });
+  }
+
+
+
```
Comment (timdh94) [inline Client/components/profile/curatedTrophies.tsx:25]: three new lines? really?

File: Client/components/profile/profileHeader.tsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {View, Text, Image, StyleSheet} from 'react-native';
+import { user } from '../../screens/ProfileScreen';
+import CircularProgressBar from './progressCircle';
+
+
+const ProfileHeader = () => {
+
+
```
Comment (timdh94) [inline Client/components/profile/profileHeader.tsx:9]: i love white space

**Review by timdh94 — COMMENTED** [review]
its okay...

File: Client/components/profile/QuestCard.tsx (hunk 5 above)
Comment (schtvr) [reply Client/components/profile/QuestCard.tsx:44]: k delete it then, ya doof

**Pull request merged**